src/components/enterprise-subsidy-requests/data/service.js

```diff
@@ -1,10 +1,12 @@
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 import { getConfig } from '@edx/frontend-platform/config';
 
-export function fetchSubsidyRequestConfiguration(enterpriseUUID) {
+export function fetchSubsidyRequestConfiguration(enterpriseUUID, useCache = true) {
   const config = getConfig();
   const url = `${config.ENTERPRISE_ACCESS_API_BASE_URL}/api/v1/customer-configurations/${enterpriseUUID}/`;
-  return getAuthenticatedHttpClient().get(url);
+  return getAuthenticatedHttpClient({
```

Comment:
[clarification] Just to confirm, this API call is safe to cache on the client for ~5 mins since we don't expect customer-configurations for a given customer to be frequently changing?

Reply:
yeah this configuration should not be changing frequently once created.